spring-kafka-test/src/main/java/org/springframework/kafka/test/assertj/KafkaConditions.java

```diff
@@ -50,6 +51,15 @@ private KafkaConditions() {
 	}
 
 	/**
+	 * @param type the type of timestamp
+	 * @param value the timestamp.
+	 * @return a Condition that matches the timestamp value in a consumer record.
+	 */
+	public static Condition<ConsumerRecord<?, ?>> timestamp(TimestampType type, long value) {
```

Comment:
I think we can add an overridden variant without `TimestampType`.
By default it is `message.timestamp.type=CreateTime`: https://cwiki.apache.org/confluence/display/KAFKA/KIP-32+-+Add+timestamps+to+Kafka+message#KIP-32-AddtimestampstoKafkamessage-Brokerconfigurationchange

But yeah, we can delegated to this one!

Also your name to the `@author`, plus we prefer to update Copyright to the actual year.

And I guess similar trick should be done for the `KafkaMatchers`.

That is good catch, @bijukunjummen , BTW!

We might have missed already a lot similar Kafka properties for testing matchers and conditions 😄